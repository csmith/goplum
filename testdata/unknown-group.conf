alert debug.sysout "debug" {}

check debug.random "test" {
  groups = ["nonexistent-group"]
}