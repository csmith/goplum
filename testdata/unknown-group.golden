"error configuring check test: no group named 'nonexistent-group'"